src/main/java/com/revealprecision/revealserver/fhir/servlet/FHIRRestfulServer.java

```diff
@@ -0,0 +1,48 @@
+package com.revealprecision.revealserver.fhir.servlet;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.server.RestfulServer;
+import com.revealprecision.revealserver.fhir.providers.CodeSystemResourceProvider;
+import com.revealprecision.revealserver.fhir.providers.LocationResourceProvider;
+import com.revealprecision.revealserver.fhir.providers.PersonResourceProvider;
+import com.revealprecision.revealserver.fhir.providers.PlanResourceProvider;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+@WebServlet(name = "fhir-facade", urlPatterns = {"/fhir/*"})
+@Component
+public class FHIRRestfulServer extends RestfulServer {
+
+  @Autowired
```

Comment:
as a suggestions prefer constructor wiring

Reply:
will fix this